src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -170,7 +173,9 @@ export default class ProcessConverter extends AbstractConverter<Process> {
       const shapeBpmnElement = findFlowNodeBpmnElement(flowNodeRef);
       const laneId = lane.id;
       if (shapeBpmnElement) {
-        shapeBpmnElement.parentId = laneId;
+        if (shapeBpmnElement.kind !== ShapeBpmnElementKind.EVENT_BOUNDARY) {
```

Comment:
:question: why not using `ShapeUtil.isBoundaryEvent` for consistency with the rest of the processing done in this class?